1.7/administration/installing/custom/advanced.md

```diff
@@ -62,6 +62,8 @@ The DC/OS installation creates these folders:
 
     **Important:** The IP address of a node must not change after DC/OS is installed on the node. For example, the IP address must not change when a node is rebooted or if the DHCP lease is renewed. If the IP address of a node does change, the node must be wiped and reinstalled.
 
+    **Important:** The IP detection scripts must begin with `#!/bin/sh` or `#!/usr/bin/env bash` as shown in the examples below. Do not add any comments before that.
```

Comment:
Thanks for shedding some light in the technicality of the issue. I'll make the changes as proposed and push.